src/public/scss/temp.scss

```diff
@@ -213,4 +213,14 @@ hr {
 
 hr.dotted {
   border-top: 4px dotted;
+}
+
+.u-align {
```

Comment:
lets give it a more accurate class name. `bg-color-float` ?

Reply:
You got it.